dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -797,8 +801,9 @@ export function PkgRepoForm(props: IPkgRepoFormProps) {
                       {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
                       <label>Package Storage Type</label>
                       <CdsControlMessage>Select the package storage type.</CdsControlMessage>
-                      {(plugin?.name === (PluginNames.PACKAGES_HELM as string) ||
-                        plugin?.name === (PluginNames.PACKAGES_FLUX as string)) && (
+                      {(
+                        [PluginNames.PACKAGES_HELM, PluginNames.PACKAGES_FLUX] as string[]
+                      ).includes(plugin?.name) && (
```

Comment:
[Not related to this PR] If in the future we want a really decoupled frontend, plugins themselves should provide this kind of information (e.g. some endpoint informing that Kapp allows [A,B,C] as package storage types).

Reply:
..or implementing the plugins customizable UI proposal 😄

Reply:
100% agree with the comment. At least, we could start extracting each param to a separate fn like a `getPluginsSupporting(foo)` that, given an input id, returns the list of plugins in which this `id` should be rendered.